app/src/main/scala/org/alephium/explorer/persistence/schema/CustomGetResult.scala

```diff
@@ -279,6 +297,36 @@ object CustomGetResult {
         eventOrder = result.<<
       )
 
+  val fungibleTokenMetadataGetResult: GetResult[FungibleTokenMetadata] =
+    (result: PositionedResult) =>
+      FungibleTokenMetadata(
+        token = result.<<,
+        symbol = result.<<,
+        name = result.<<,
```

Comment:
The inserted data for `symbol` and `name` gets swapped on query. These need to be swapped here to match the table's column order: `token, name, symbol, decimal`. 

```scala
token = TokenId(...)
symbol = "name"
name = "symbol"
decimals = "0"
```

Reply:
OMG very very nice and critical catch, good review job!

Reply:
fixed here [faba641](https://github.com/alephium/explorer-backend/pull/486/commits/faba6410f7fbbe7324ba1236e90c73351239e9cc)